ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx

```diff
@@ -0,0 +1,513 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  SmartTransactionStatuses,
+  SmartTransaction,
+} from '@metamask/smart-transactions-controller/dist/types';
+
+import {
+  Box,
+  Text,
+  Icon,
+  IconName,
+  IconSize,
+  Button,
+  ButtonVariant,
+  ButtonSecondary,
+} from '../../../components/component-library';
+import {
+  AlignItems,
+  BlockSize,
+  BorderStyle,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextVariant,
+  TextColor,
+  FontWeight,
+  IconColor,
+  TextAlign,
+} from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getCurrentChainId } from '../../../selectors';
+import { getFeatureFlagsByChainId } from '../../../../shared/modules/selectors';
+import { BaseUrl } from '../../../../shared/constants/urls';
+import { hideLoadingIndication } from '../../../store/actions';
+import { hexToDecimal } from '../../../../shared/modules/conversion.utils';
+
+type RequestState = {
+  smartTransaction?: SmartTransaction;
+  isDapp: boolean;
+};
+
+export type SmartTransactionStatusPageProps = {
+  requestState: RequestState;
+  onCloseExtension: () => void;
+  onViewActivity: () => void;
+};
+
+const SMART_TRANSACTIONS_EXPECTED_DEADLINE_FALLBACK = 45;
+const SMART_TRANSACTIONS_MAX_DEADLINE_FALLBACK = 150;
```

Comment:
What's the difference between these and the ones in `shared/constants/smartTransactions.ts`

Reply:
I've moved these into /smartTransactions.ts and renamed them.